build/containers/Dockerfile.agent

```diff
@@ -9,7 +9,16 @@ RUN echo "Using Rust binaries from ${CROSS_BUILD_TARGET}/${BUILD_TYPE}"
 # Link the container to the Akri repository
 LABEL org.opencontainers.image.source https://github.com/project-akri/akri
 
-RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl && apt-get clean
+RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl curl ca-certificates && apt-get clean
+
 COPY ./target/${CROSS_BUILD_TARGET}/${BUILD_TYPE}/agent /agent
+
+# Install crictl
+RUN VERSION="v1.17.0" && \
```

Comment:
Why did you pick 1.17? We had problems using older crictl against newer clusters.

Reply:
This is the version we have been using in our e2e tests and [documentation](https://docs.akri.sh/user-guide/cluster-setup#configure-crictl). 

>We also may want to consider using a more recent version of crictl and add updating it to our release cycle.

I think our next release we can consider moving to a newer version